react-ui/src/components/Layouts/Dashboard.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component, Fragment } from "react";
```

Comment:
Hi Kyle,
This file was removed in a past PR. When updating your branch with master, make sure to resolve conflicts appropriately. My tip would be to look at the files you changed on your PR to make sure there was no undesirable changes in your PR.